src/elm/quantity.coffee

```diff
@@ -274,3 +280,6 @@ module.exports.doDivision = (a,b) ->
 
 module.exports.doMultiplication = (a,b) ->
   if a instanceof Quantity then a.multiplyBy(b) else b.multiplyBy(a)
+
+coalesceToOne = (o) ->
+  if o? then o else '1'
```

Comment:
As noted above, should we also be checking for empty string (`""`)?  I _think_ so?